docs/installing.rst

```diff
@@ -27,21 +28,54 @@ In your ``conf.py`` file:
     :ref:`supported-dependencies`
         Officially Supported versions of Python, Sphinx, and other dependencies.
 
+
 .. note::
 
    Adding this theme as an extension is what enables localization of theme
    strings in your translated output. If these strings are not translated in
    your output, either we lack the localized strings for your locale, or you
    are using an old version of the theme.
 
+   ..
+      comment about this note: it's possibly not necessary to add the theme as an extension.
+      Rather, this is an issue caused by setting html_theme_path.
+      See: https://github.com/readthedocs/readthedocs.org/pull/9654
+
+
+.. _howto_upgrade:
+
+How to upgrade
+--------------
+
+.. tip:: 
+    **Read the Docs users:** To know which version of ``sphinx-rtd-theme`` you are running, you can always check your latest build outputs. One or more steps in the documentation build will install python packages, one of which is ``sphinx-rtd-theme``.
```

Comment:
Perhaps this note should go in RTD docs instead, this isn't specific to just this theme.

Reply:
It's one of the few packages that we install implicitly, so I think it's important to let people know about understanding when it's installed and how to inspect that.

Reply:
The purpose of a "tip" box is to be able to have a bit of information that isn't weaved into the surrounding parts? Isn't this a good case for a tip?

> "Check one or more steps in your build" is probably lost on a user that is not familiar with RTD.

Hence, the tip box was clearly labeled for "Read the Docs users:" :)

I moved it to the bottom of the chapter, maybe that's better?

Reply:
The directions here now aren't specific enough to help users that don't already know what they are looking for -- what steps are they looking at, what output are they looking for, etc? There's a lot more here, but it's fairly out of scope for these docs.

Reply:
It's not out of scope. See for instance: https://stackoverflow.com/a/74355734/405682

By giving people some pointers to understand the RTD build, they can solve issues like this. I'm gonna go ahead and improve the tip box :)

Reply:
I haven't been talking about a pointer being out scope. Covering how to use Read the Docs in the theme docs *is* out of scope.

Again, the docs you are describing belong in the application docs, not here. Adding a pointer to relevant docs in our application is absolutely fine, that's not in question though.